services/nutritional/nutritional_job_service.py

```
@@ -1,89 +1,174 @@
 
-scheduler = BackgroundScheduler()
 
+def _get_active_schemas() -> list:
+    """Return all distinct schemas with at least one active user.
+
+    Queries public.usuario — always accessible without schema_translate_map
+    because User.__table_args__ has schema='public'.
+    """
+    rows = (
+        db.session.query(User.schema)
+        .filter(User.active == True)
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `True`; use `User.active:` for truth checks

Replace with `User.active`

(E712)